pytest_tests/steps/cluster_test_base.py

```diff
@@ -17,18 +19,24 @@ def fill_mandatory_dependencies(self, cluster: Cluster, client_shell: Shell):
         yield
 
     @allure.title("Tick {epochs_to_tick} epochs")
-    def tick_epochs(self, epochs_to_tick: int):
+    def tick_epochs(self, current_epoch: int, epochs_to_tick: int):
```

Comment:
This method looks to me like tick_epochs_and_wait. Do we really need the second method? We could use self.get_epoch() as a default here if current_epoch is not specified